src/System.IO.UnmanagedMemoryStream/tests/UmsFlush.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.IO.Tests
 {
-    public class FlushTests
+    public partial class FlushTests
```

Comment:
This will be potentially problematic. Please upstream the change with better test name

Reply:
Done, our PR is https://github.com/mono/corefx/pull/176